packages/babel-preset-env/data/plugin-features.js

```diff
@@ -97,10 +97,12 @@ const es = {
 
   "proposal-json-strings": "JSON superset",
   "proposal-optional-catch-binding": "optional catch binding",
+  "proposal-optional-chaining": "optional chaining operator (?.)",
   "transform-named-capturing-groups-regex": "RegExp named capture groups",
   "transform-member-expression-literals": "Object/array literal extensions / Reserved words as property names",
   "transform-property-literals": "Object/array literal extensions / Reserved words as property names",
   "transform-reserved-words": "Miscellaneous / Unreserved words",
+  "proposal-nullish-coalescing-operator": "nullish coalescing operator (??)",
```

Comment:
Can you keep this alphabetically ordered?

Reply:
Not a blocker for this PR, but would be nice if we can enforce this with ESLint, even if it's just enabling the rule using comments for this object.

Reply:
Yeah that would be cool! But let's do it in a separate PR.

Reply:
Ouch - I didn't see this reply until after making the changes. I selectively added the `sort-keys` rule to a few files which resulted in some noise/sorting of existing keys. It's in a separate commit for easy review. If it's too much, I can pull it out into another PR.

Reply:
It's still okay to review since the changes are related to a few files.